helios-services/src/main/java/com/spotify/helios/agent/HealthCheckerFactory.java

```diff
@@ -90,14 +93,17 @@ public boolean check(final String containerId) {
           healthCheck.getCommand().toArray(new String[healthCheck.getCommand().size()]);
 
       try {
-        final String execId = docker.execCreate(
+        final ExecCreation execCreation = docker.execCreate(
             containerId, cmd,
             DockerClient.ExecCreateParam.attachStdout(),
             DockerClient.ExecCreateParam.attachStderr());
+        final String execId = execCreation.id();
 
-        final String output;
+        String output = "";
         try (LogStream stream = docker.execStart(execId)) {
           output = stream.readFully();
+        } catch (IOException e) {
+          // thrown from implicit call to close(). Ignore.
```

Comment:
why do we ignore this exception? 

fwiw it seems like docker-client's DefaultLogStream.close() can not throw an IOException, so perhaps the LogStream interface should redefine close() to not throw a checked exception.